content/applications/inventory_and_mrp/inventory/product_management/product_tracking/packaging.rst

```diff
@@ -140,4 +140,50 @@ used for the product. The chosen packaging is displayed on the :abbr:`SO (Sales
       :align: center
       :alt: Assign packagings on the Sales Order Line.
 
+.. _inventory/product_management/packaging-route:
+
+Routes for packaging
+====================
+
+When receiving packagings, by default, they follow the warehouse's :doc:`configured reception route
+<../../shipping_receiving/daily_operations/shipments_deliveries>`. To **optionally** set up a
+packaging-specific route, go to :menuselection:`Inventory app --> Configuration --> Routes`.
+
+.. important::
+   The :guilabel:`Product Packagings`, :guilabel:`Storage Locations`, and :guilabel:`Multi-Step
+   Routes` features (found by going to :menuselection:`Inventory app --> Configuration -->
```

Comment:
```suggestion
   The *Product Packagings*, *Storage Locations*, and *Multi-Step
   Routes* features (found by going to :menuselection:`Inventory app --> Configuration -->
```
opting for italics here because these features aren't being shown on the screen at the moment. Currently, if the reader was following along, they'll be on the 'Routes' page